optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/test/java/org/optaplanner/quarkus/constraints/OptaPlannerProcessorConstraintProviderAndDrlTest.java

```
@@ -33,14 +45,195 @@
+                                    return false;
+                                }
+                                Path p = Paths.get(localRepo);
+                                for (String s : artifact.getGroupId().split("\\.")) {
+                                    p = p.resolve(s);
+                                }
+                                p = p.resolve(artifact.getArtifactId());
+                                p = p.resolve(artifact.getVersion());
+                                p = p.resolve(LocalWorkspace.getFileName(artifact));
+                                return Files.exists(p);
+                            }
+
+                            @Override
+                            public List<String> findVersions(Artifact artifact) {
+                                return ws.findVersions(artifact);
+                            }
+                        });
+
+                        final MavenArtifactResolver resolver = MavenArtifactResolver.builder()
+                                .setRepositorySystem(mvnCtx.getRepositorySystem())
+                                .setRepositorySystemSession(session)
+                                .setRemoteRepositories(mvnCtx.getRemoteRepositories())
+                                .setRemoteRepositoryManager(mvnCtx.getRemoteRepositoryManager())
+                                .setCurrentProject(currentProject)
+                                .build();
+                        System.out
+                                .println("  " + resolver
+                                        .resolve(new DefaultArtifact(currentProject.getGroupId(),
+                                                currentProject.getArtifactId(), "jar", currentProject.getVersion()))
+                                        .getArtifact().getFile());
+                    } catch (BootstrapMavenException e) {
+                        // TODO Auto-generated catch block
+                        e.printStackTrace();
```

don't eat exceptions